portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -27,18 +27,21 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     UserService userService = UserServiceFactory.getUserService();
 
     List<String> list = new ArrayList<>();
+
+    list.add(String.valueOf(userService.isUserLoggedIn()));
+
     if (userService.isUserLoggedIn()) {
       String userEmail = userService.getCurrentUser().getEmail();
       String urlToRedirectToAfterUserLogsOut = "/comments.html";
```

Comment:
[Optional] It would be fine to hard pass this directly as a constant into the function below rather than creating a variable that is used only once.

Same for 'urlToRedirectToAfterUserLogsIn' below.